pkg/controller/build/buildrequest/assets/Containerfile.on-cluster-build-template

```diff
@@ -11,6 +11,9 @@ RUN mkdir -p /etc/machine-config-daemon && \
 	cat /tmp/machineconfig.json.gz | base64 -d | gunzip - > /etc/machine-config-daemon/currentconfig
 
 FROM {{.BaseOSImage}} AS configs
+RUN dnf install -y fuse-overlayfs && \
+    mkdir -p /etc/containers && \
+    echo -e "[storage]\ndriver = \"overlay\"\n\n[storage.options]\nmount_program = \"/usr/bin/fuse-overlayfs\"" > /etc/containers/storage.conf
```

Comment:
Hmm, this will configure fuse-overlayfs on the CoreOS node itself, which we definitely don't want to do. I think instead, you probably meant to configure this in the environment in which `buildah bud` is being run? (One level up.)